src/actions/user.js

```diff
@@ -7,6 +7,9 @@ export default function enrichUserData() {
   return (dispatch, getState) => {
     dispatch(createAction('fetchUserData')());
     const state = getState();
+    if (!state.oidc.user) {
+      return null;
+    }
     const url = `v1/users/${state.oidc.user.profile.sub}`;
     return apiGet(state, url).then((democracyUser) => democracyUser.json()).then((democracyUserJSON) => {
```

Comment:
Could add error handling here so that header does not end up in broken state. Something like:

`const url = `v1/users/${state.oidc.user.profile.sub}`;

    try {
      const response = await apiGet(state, url);

      if (response.status === 401) {
        throw new Error('Unauthorized');
      }

      const json = await response.json();

      const userWithOrganization = {
        displayName: `${get(json, 'first_name')} ${get(json, 'last_name')}`,
        nickname: get(json, 'nickname'),
        answered_questions: get(json, 'answered_questions'),
        favorite_hearings: get(json, 'followed_hearings'),
        adminOrganizations: get(json, 'admin_organizations', []),
        hasStrongAuth: get(json, 'has_strong_auth', false)
      };

      return dispatch(createAction('receiveUserData')(userWithOrganization));
    } catch (error) {
      return dispatch(createAction('clearUserData'));
    }`